streams/quickstart/java/src/main/resources/archetype-resources/src/main/java/LineSplit.java

```diff
@@ -44,7 +44,7 @@ public static void main(String[] args) throws Exception {
 
         final StreamsBuilder builder = new StreamsBuilder();
 
-        builder.stream("streams-plaintext-input")
+        builder.<String, String>stream("streams-plaintext-input")
```

Comment:
This fix should actually be provided by https://github.com/apache/kafka/pull/6194 -- I included it here to let the build pass.